src/test/suite/fixtures/recorded/languages/rust/chuckValueFine.yml

```diff
@@ -0,0 +1,38 @@
+languageId: rust
+command:
+  spokenForm: chuck value fine
+  version: 2
+  targets:
+    - type: primitive
+      mark: {type: decoratedSymbol, symbolColor: default, character: f}
+      modifiers:
+        - type: containingScope
+          scopeType: {type: value}
+  usePrePhraseSnapshot: true
+  action: {name: remove}
+initialState:
+  documentContents: |
+    let user = User {
+        first_name: "John",
+        last_name: "Doe",
+    };
+  selections:
+    - anchor: {line: 3, character: 2}
+      active: {line: 3, character: 2}
+  marks:
+    default.f:
+      start: {line: 1, character: 4}
+      end: {line: 1, character: 14}
+finalState:
+  documentContents: |
+    let user = User {
+        first_name,
```

Comment:
Just to check: is this valid code?

Reply:
Perfectly valid assuming there's a `first_name` variable in scope.

Reply:
Got it.  So in that case, I'd remove the `:` for chuck key / name and chuck value.  We do that kinda thing all the time in typescript; quite convenient